model/constants.py

```diff
@@ -110,7 +122,7 @@ class VSSDataType(Enum):
     INT32 = "Int32"
     UINT32 = "UInt32"
     INT64 = "Int64"
-    UINT64 = "UInt64"
+    UINT64 = "Int64"
```

Comment:
?

Reply:
Not sure what the question here is, can you elaborate?

Reply:
The question is why UINT is mapped to INT (with no explanation).
However, it might be related to this? 
https://github.com/GENIVI/vehicle_signal_specification/issues/163

Reply:
... or the question could be why you changed something at all in this file.  It seems unrelated to the main topic here?